server/src/main/java/com/objectcomputing/checkins/services/memberprofile/currentuser/CurrentUserDTO.java

```diff
@@ -0,0 +1,133 @@
+package com.objectcomputing.checkins.services.memberprofile.currentuser;
+
+import io.micronaut.core.annotation.Introspected;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.annotation.Nullable;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Past;
+import java.time.LocalDate;
+import java.util.UUID;
+
+@Introspected
+public class CurrentUserDTO {
+
+    @NotNull
+    @Schema(description = "id of the member profile this entry is associated with", required = true)
+    private UUID id;
+
+    @NotBlank
+    @Schema(description = "full name of the employee", required = true)
+    private String name;
+
+    @Nullable
+    @Schema(description = "employee's role at the company", required = true)
+    private String role ;
+
+    @Nullable
+    @Schema(description = "employee's professional development lead")
+    private UUID pdlId;
+
+    @Nullable
+    @Schema(description = "where the employee is geographically located", required = true)
+    private String location;
+
+    @NotBlank
+    @Schema(description = "employee's OCI email. Typically last name + first initial @ObjctComputing.com", required = true)
+    private String workEmail;
+
+    @Nullable
+    @Schema(description = "unique identifier for this employee with the Insperity system")
+    private String insperityId;
+
+    @Nullable
+    @Past
+    @Schema(description = "employee's date of hire", required = true)
+    private LocalDate startDate;
+
+    @Nullable
+    @Schema(description = "employee's biography")
+    private String bioText;
+
+    @Nullable
+    @Schema(description = "Image URL of the user")
+    private String imageUrl;
```

Comment:
Added ImageUrl to the DTO. This url isn't being saved to the DB. (So if a user changes his/her objectcomputing.com profile picture, it should automatically change in our system as well)

Reply:
We need to add this eventually to the actual MemberProfile. Otherwise we won't be able to populate the images for other user profiles on the website. I think we should just make that change now, or make plans to do so. @mkimberlin thoughts?

Reply:
I'm working on that issue, actually. I was thinking I might drag you guys into a conversation about it tomorrow morning with Chuck. Basically, I asked that the application service user be allowed to access the directory API for users:

https://developers.google.com/admin-sdk/directory/v1/reference/users/